endpointmanager/pkg/endpointmanager/fhirendpoint.go

```diff
@@ -35,6 +35,7 @@ type FHIREndpoint struct {
 // FHIREndpointStore is the interface for interacting with the storage layer that holds
 // FHIR endpoint objects.
 type FHIREndpointStore interface {
+	GetAllFHIREndpoints(ctx context.Context) (*[]FHIREndpoint, error)
```

Comment:
check out this: https://blog.golang.org/slices-intro

this is a slice, which is a reference type, pointing to the underying array. Because of this, it's efficient to pass by value, so you don't need to return a reference to it here, which will simplify some of your other code which needs to continually dereference it.

Reply:
ah okay, i didn't realize that's how slices work!